collect_app/src/androidTest/java/org/odk/collect/android/support/pages/NotificationDrawer.kt

```diff
@@ -99,6 +99,12 @@ class NotificationDrawer {
             // "Clear all" doesn't exist because there are notifications to clear - just press back
             pressBack()
         }
+
+        // Exit with back if "Clear all" didn't take care of it
+        if (device.findObject(By.text("Notifications")) != null) {
+            device.findObject(By.text("Clear all"))
```

Comment:
We do the same above so why do we need this here?

Reply:
Oh no. I had meant this to call `pressBack` in the `if`, but I made a mistake, and somehow it solved the problem I was having. Potentially the extra element look-ups delay other assertions long enough to fix something. I'll investigate further.